roles/KubevirtNodeLabeller/templates/kubevirt-node-labeller-ds-v0.0.5.yaml

```diff
@@ -18,12 +18,12 @@ spec:
       serviceAccount: kubevirt-node-labeller
       containers:
       - name: kubevirt-node-labeller-sleeper
-        image: {{ kubevirt_node_labeller }}
+        image: "{{ repo_info['registry'] + '/' + kubevirt_node_labeller_image + ':' + kubevirt_node_labeller_tag }}"
```

Comment:
Please create an ssp_registry variable intead of using the ["registry"] references. Then you could do for example `{{ ssp_registry |default("kubevirt") }}/{{ image }}:{{ tag }}` and kubevirt will be used for upstream where the role was disabled (for example)

Reply:
Good idea, done - also set backward-compatible defaults for all images